skype/skype-ps/skype/New-CsOnlineLisLocation.md

```diff
@@ -25,7 +23,7 @@ New-CsOnlineLisLocation -Location <string> -CivicAddressId <guid> [-CityAlias <s
 [-Elin <string>] [-Force] [-HouseNumberSuffix <string>] [-Latitude <string>] [-Longitude <string>] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
-### CreateCivicAddress
+### CreateCivicAddress (This parameter set variant is not supported and will be deprecated)
```

Comment:
This text will not be shown in the published version of the article. I think it will be better to add the unsupported parameters to the SYNOPSYS section

Reply:
I see you added comments about support status to every parameter, so that should cover it